core/src/main/java/nu/yona/server/subscriptions/service/UserDto.java

```diff
@@ -189,7 +189,6 @@ void updateUser(User originalUserEntity)
 		originalUserEntity.setLastName(lastName);
 		originalUserEntity.setMobileNumber(mobileNumber);
 		originalUserEntity.setNickname(privateData.getNickname());
-		originalUserEntity.setUserPhotoId(privateData.getUserPhotoId());
```

Comment:
Oops... I think this is a left-over from some earlier ideas on how to do user photo updates.